mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -168,8 +168,13 @@ func resourceMongoDBAtlasCluster() *schema.Resource {
 				Computed: true,
 			},
 			"provider_encrypt_ebs_volume": {
+				Type:       schema.TypeBool,
+				Optional:   true,
+				Deprecated: "All EBS volumes are now encrypted by default, no option to have encryption disabled",
```

Comment:
Let's change this a bit to reduce confusion: 
"All EBS volumes are encrypted by default, the option to disable encryption has been removed",